plugins/BEdita/API/src/Controller/AdminController.php

```diff
@@ -32,6 +32,11 @@ class AdminController extends ResourcesController
     public $modelClass = null;
 
     /**
+     * {@inheritDoc}
+     */
+    protected $_routeNamePrefix = 'api:admin';
```

Comment:
why not `protected $routeNamePrefix`? In general we avoid to use `_` except for CakePHP things

Reply:
yes, you're right

Reply:
done